README.md

```diff
@@ -67,9 +67,11 @@ Push changes to the staging site with docker-compose:
 
 ```bash
 # staging
+$ export GIT_COMMIT="$(git rev-parse --short HEAD)"
```

Comment:
I think we can get rid of any references to `GIT_COMMIT` in the `README` and `scripts/infra`